src/selectedtests/datasource/mongo_wrapper.py

```diff
@@ -1,6 +1,8 @@
 """Classes for accessing mongo collections."""
+from __future__ import annotations
```

Comment:
It looks like the Dockerfile for this projects is using the ubuntu package python3, which I think is python 3.6. This annotation was added in python 3.7. So I think you will need to update the Dockerfile to to use at least python 3.7 for this to work (I recently hit this with the build-baron-tools). The easiest way to do that would be to start basing this image off the python 3.7 debian image.

Reply:
Created a ticket for this: https://jira.mongodb.org/browse/MAKE-1075. For now I removed annotations.